public/app/features/transformers/editors/GroupByTransformerEditor.tsx

```diff
@@ -49,8 +49,28 @@ export const GroupByTransformerEditor = ({
     [onChange]
   );
 
+  // See if there's both an aggregation and grouping field configured
+  // for calculations. If not we display a warning because there
+  // needs to be a grouping for the calculation to have effect
+  let hasGrouping,
+    hasAggregation = false;
+  for (const field of Object.values(options.fields)) {
+    if (field.aggregations.length > 0 && field.operation !== null) {
+      hasAggregation = true;
+    }
+    if (field.operation === GroupByOperationID.groupBy) {
+      hasGrouping = true;
+    }
+  }
+  const showCalcAlert = hasAggregation && !hasGrouping;
+
+  console.log(options);
```

Comment:
Is this `console.log()` supposed to be here?

Reply:
ah good catch, i'll push up a commit with a bit of code cleanup 👍